coursework-template.Rmd

```diff
@@ -179,37 +178,72 @@ You could pick one of these topics:
-1. Description of visualisation techniques, e.g. with reference to particular packages, commands, or the literature on how to produce effective data visualisations
+1. High quality, attractive or advanced techniques (e.g. multi-layered maps or graphs, facets or other advanced techniques)
 1. Using visualisation techniques appropriate to the topic and data and interpreting the results correctly (e.g. mentioning potential confounding factors that could account for observed patterns)
+1. The report is well-formatted, accessible (e.g. with legible text size and does not contain excessive code in the submitted report) and clearly communicates the data and analysis visually, with appropriate figure captions, cross-references and a consistent style
+
+**Distinction (70%+):** The report contains high quality, attractive, advanced and meaningful visualisations that are well-described and interpreted correctly. The report is well-formatted, accessible and clearly communicates the data and analysis visually.
+
+**Merit (60-69%):** Between pass and distinction.
+
+**Pass (50-59%):** The report contains basic visualisations or are not well-described or interpreted correctly or the report is poorly formatted, not accessible or does not clearly communicate the data and analysis visually.
+
+**Fail (0-49%):** The report is of unacceptable quality (would likely be rejected in a professional setting) and/or has poor quality and/or few visualisations, or the visualisations are inappropriate given the data and research questions.
+
+    **Code quality, efficiency and reproducibility: 20%**
+
+1. Code quality in the submitted source code, including using consistent style, appropriate packages, and clear comments
+1. Efficiency, including pre-processing to reduce input datasets (avoiding having to share large datasets in the submission for example) and computationally efficient implementations
+1. The report is fully reproducible, including generation of figures. There are links to online resources for others wanting to reproduce the analysis for another area, and links to the input data
+
+**Distinction (70%+):** The source code underlying the report contains high quality, efficient and reproducible code that is well-written, using consistent syntax and good style, well-commented and uses appropriate packages. The report is fully reproducible, with links to online resources for others wanting to reproduce the analysis for another area, and links to the input data.
+
+**Merit (60-69%):** Between pass and distinction.
+
+**Pass (50-59%):** The source code underlying the report is available and describes the outputs in the report but is not well-commented, not efficient or has very limited levels of reproduicibility, with few links to online resources for others wanting to reproduce the analysis for another area, and few links to the input data.
+
+**Fail (0-49%):** The report has little to no reproducible, readable or efficient code.
```

Comment:
So a student that did a first class piece of work but forgot to submit the code would scrape a pass. 0.8 × 70% = 56%? In clear on how to award grades between 0 and 49%.